ocpp/v16/call.py

```diff
@@ -229,3 +234,72 @@ class DataTransferPayload:
     vendor_id: str
     message_id: Optional[str] = None
     data: Optional[str] = None
+
+
+# Backported CALL Messages
+# Improved security for OCPP 1.6-J specification
+
+@dataclass
+class CertificateSignedPayload:
+    cert: List
+
+
+@dataclass
+class DeleteCertificatePayload:
+    certificate_hash_data: Dict
+
+
+@dataclass
+class ExtendedTriggerMessagePayload:
+    requested_message: MessageTriggerType
```

Comment:
the field `connetor_id` is missing. Please add it.

![Screenshot_2021-04-26_11-19-21](https://user-images.githubusercontent.com/1565144/116059572-44313680-a681-11eb-871f-785dd6c5bca3.png)